runtime/Makefile

```diff
@@ -5,8 +5,8 @@ $(error Unsupported Architecture. Supports x86_64 and aarch64, found $(ARCH))
 endif
 endif
 
-TOTAL_CORES := $(shell getconf _NPROCESSORS_CONF)
-PAGE_SIZE := $(shell getconf PAGESIZE)
+# TOTAL_CORES := $(shell getconf _NPROCESSORS_CONF)
+# PAGE_SIZE := $(shell getconf PAGESIZE)
```

Comment:
Can you please uncomment this? I don't see this defined anywhere else, and we do use this for guard pages and such.

For example:
```c
void *addr = mmap(NULL, sandbox_size + linear_memory_max_size + /* guard page */ PAGE_SIZE, PROT_NONE,
	                  MAP_PRIVATE | MAP_ANONYMOUS, -1, 0);
```